dotnet/Examples/TokenAuth.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.IO;
+using VaultSharp;
+using VaultSharp.V1.AuthMethods;
+using VaultSharp.V1.AuthMethods.Token;
+using VaultSharp.V1.Commons;
+
+// WARNING: The code in this hello-world file is potentially insecure. It is not safe for use in production.
+// This is just a quickstart for trying out the Vault client library for the first time.GetSecretWithToken()
```

Comment:
This comment accidentally has GetSecretWithToken() shoved onto the end

Reply:
thanks for the catch!